src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,17 @@
 
 public class MainApp {
 
+  Machine excavator = new Excavator();
+  Machine bulldozer = new Bulldozer();
+  Machine truck = new Truck();
```

Comment:
It's recommended to use abstract references instead of specific ones where possible. For example, instead of 'Machine excavator = new Excavator();' you should use 'Machine machine1 = new Excavator();'. This allows for more flexible code that can handle different types of machines without modification.